.travis.yml

```diff
@@ -64,25 +65,33 @@ addons:
     # https://scan.coverity.com/travis_ci
     project:
       name: "JanusGraph/janusgraph"
-      version: "0.2.0-SNAPSHOT"
```

Comment:
Not sure why this line was removed; this identifies the build on the Coverity dashboard, so that we can see what version of JanusGraph the given Coverity scan refers to. It appears that it'll need to be updated manually, but it doesn't seem like there is any other way.

Alternatively, we can remove this entire stanza and use the script I wrote separately to submit the build for Coverity to scan, which also knows how to parse `pom.xml` to retrieve the current version automatically. I'll look into this later.